lib/csvlint/csvw/property_checker.rb

```diff
@@ -66,7 +66,7 @@ def check_common_property_value(value, base_url, lang)
                   raise Csvlint::Csvw::MetadataError.new(), "common property with @value has properties other than @language or @type" unless value.except("@type").except("@language").except("@value").empty?
                 when "@language"
                   raise Csvlint::Csvw::MetadataError.new(), "common property with @language lacks a @value" unless value["@value"]
-                  raise Csvlint::Csvw::MetadataError.new(), "common property has invalid @language (#{v})" unless v =~ BCP47_LANGUAGE_REGEXP || v.nil?
```

Comment:
so stuff like `v =~ /some.regex/` works in ruby but when you use a string variable you should be wrapping it in `Regex.new()` i guess?

Reply:
Yes, so that `BCP47_LANGUAGE_REGEXP` was a complicated concatenated string variable. To use it we have to say that is a regexp or may be define it inline.

The other main regex used in this class `BCP47_REGEXP` was initialised as regex. I think with this variable they kind of forgot to do that and since this part of code is executed very rarely it went unnoticed.

Reply:
Static analysis FTW!